README.md

```diff
@@ -349,16 +357,16 @@ node scripts/updateVersion.js
 Mark the new version with Lerna:
 
 ```sh
-lerna version 24.0.0-alpha0 --no-push --no-git-tag-version --yes
+lerna version 24.0.0-alpha0 --no-push --no-git-tag-version --force-publish --exact --yes
```

Comment:
Added `--force-publish` flag to enforce increasing version, otherwise Lerna does nothing with the notice:

```
lerna notice Current HEAD is already released, skipping change detection.
```

Also, `--exact` flag is needed to pin dependencies versions during alpha / beta stage.